core/src/main/scala/org/apache/spark/api/java/JavaSparkContext.scala

```diff
@@ -774,6 +774,16 @@ class JavaSparkContext(val sc: SparkContext)
 
   /** Cancel all jobs that have been scheduled or are running. */
   def cancelAllJobs(): Unit = sc.cancelAllJobs()
+
+  /**
+   * Returns an Java map of JavaRDDs that have marked themselves as persistent via cache() call.
+   * Note that this does not necessarily mean the caching or computation was successful.
+   */
+  def getPersistentRDDs : JMap[java.lang.Integer, JavaRDD[_]] = {
```

Comment:
Nit: remove space before colon, and remove braces on the next line. I'm sort of surprised if this works but let's see the tests.